### PR TITLE
Ernestomr87 t38 create button component

### PR DESCRIPTION
**Pull request by stdevGeekmidget**
[mobilewallet] refs#38 Create Mobile Wallet

### CODE REVIEW

**Review by samuelvisscher — CHANGES_REQUESTED** [review]
As discussed in telegram, please write the component to be in line with our current button component:

https://github.com/samuelvisscher/skycoin/tree/develop/src/gui/static/src/app/components/layout/button

**Comment by ernestomr87** [comment]
Done !!
"As discussed in telegram, please write the component to be in line with our current button component"

File: src/components/button/button.component.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import {Component, EventEmitter, Input, Output} from '@angular/core';
+
+@Component({
+  selector: 'app-button',
+  templateUrl: 'button.component.html',
+})
+
+export class ButtonComponent {
+  @Input() disabled: any;
+  @Input() error: string;
```
Comment (samuelvisscher) [inline src/components/button/button.component.ts]: Let's manage error & state similar to: https://github.com/skycoin/skycoin/blob/develop/src/gui/static/src/app/components/layout/button/button.component.ts

File: src/components/button/button.component.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import {Component, EventEmitter, Input, Output} from '@angular/core';
```
Comment (samuelvisscher) [inline src/components/button/button.component.ts:1]: add space ..

**Review by samuelvisscher — APPROVED** [review]

**Pull request merged**